backend/functions/routes/heatmaps-coordinates.js

```diff
@@ -0,0 +1,52 @@
+const {app} = require('../setups/setup.js');
+const MongoClient = require('mongodb').MongoClient;
+
+const DB = '947g5';
+const COLLECTION = 'addressbase';
+const USERNAME = 'react-read';
+const PASSWORD = '31vTs8wfBB9KvfIy';
+
+app.post('/heatmaps', (req, res) => {
+  const uri = `mongodb+srv://${USERNAME}:${PASSWORD}@cluster0-cbuck.mongodb.net/test?retryWrites=true&w=majority`;
```

Comment:
We do not have to have this URL here if we plan to use Firebase functions to call MongoDB. Also the URL needs to be encrypted.

Reply:
This is a firebase function.